src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestMetadata.cs

```diff
@@ -333,7 +333,7 @@ public IEnumerable<string> Validate()
                 yield return NuGetResources.Manifest_RequireLicenseAcceptanceRequiresLicenseUrl;
             }
 
-            if(_licenseUrl != null && LicenseMetadata != null && !_licenseUrl.Equals(LicenseMetadata.LicenseUrl))
+            if (_licenseUrl != null && LicenseMetadata != null && !new Uri(_licenseUrl).Equals(LicenseMetadata.LicenseUrl))
```

Comment:
Can't you use the public property `LicenseUrl`?

Reply:
I didn't even see that. I'll use that.